airbyte-integrations/connectors/source-monday/source_monday/extractor.py

```diff
@@ -105,9 +105,14 @@ def extract_records(self, response: requests.Response) -> List[Record]:
         if not result and self.field_path_incremental:
             result = self.try_extract_records(response, self.field_path_incremental)
 
-        for item_index in range(len(result)):
-            if "updated_at" in result[item_index]:
-                result[item_index]["updated_at_int"] = int(
-                    datetime.strptime(result[item_index]["updated_at"], "%Y-%m-%dT%H:%M:%S%z").timestamp()
-                )
+        for record in result:
+            if "updated_at" in record:
```

Comment:
If this is the `cursor` value (`updated_at`) can we take it from the manifest or declare the constant otherwise?

Reply:
Frankly speaking - I'm not sure if this is a cursor value. There has been a change regarding the way we iterate the `result`, but nothing changed regarding this specific field. If you suggest declaring it as a constant, I'd prefer doing it in a separate PR